vars/README.md

```diff
@@ -10,4 +10,16 @@ List of top level variables.
 | openshift_version | 3.7 | OpenShift cluster version. Either 3.7 or 3.9 |
 | manifest_version | release | KubeVirt manifest version |
 | docker_tag | latest | Container image tag |
-| storage_role | None | Storage role to install with KubeVirt |
+| storage_role | None | Storage role  to install with KubeVirt |
+
+
+### Storage Roles
+
+**storage-demo**
+All-in-one ephemeral storage.
+
+**CNS**
+```openshift-ansible``` will provide Gluster storage and the CNS role will
+create the storage class.
+
+**Cinder**
```

Comment:
Honestly, I don't like how it looks -how some variables go under the table, what if others appear?  And I don't like hardcodng default values in docs too. ATM I think about two solutions.
1) Can it all be in one table? with Defaults and Possible values?
2) If not, what about writing all possible values in a table in Readme and for default values add a reference to var file?

Reply:
That's fair.  I think storage-roles could be their own document frankly, but I can list the defaults in the tbale